Sources/Views/MessageInputBar.swift

```diff
@@ -136,6 +136,9 @@ open class MessageInputBar: UIView {
                 $0.messageInputBar?.didSelectSendButton()
         }
     }()
+
+    /// A boolean that determines if the sendButton's `isEnabled` state should be auto updated on text changes.
+    open var shouldAutoUpdateSendButtonEnabledStateOnTextDidChange = true
```

Comment:
I'm cool with this 😎 Thanks for making the PR. Can we rename it `shouldManageSendButtonEnabledState`? Open to shorter suggestions 😂 

We also need to add a CHANGELOG entry under Added

Reply:
I was just following the naming conventions I saw used elsewhere for the sake of being consistent. (Specifically `shouldAutoUpdateMaxTextViewHeight`). I'm up for being more concise as long as it's not at the expense of clarity! `shouldManageSendButtonEnabledState` is  great. I'll update that and add a CHANGELOG entry.

Reply:
Done 👍